hostscripts/scale-cloud/setup-baremetal.sh

```diff
@@ -13,6 +13,7 @@ export want_ldap=1
 export want_all_ssl=1
 export controller_raid_volumes=2
 export reposerver=provo-clouddata.cloud.suse.de
+export susedownload=ibs-mirror.prv.suse.net
```

Comment:
This mirror works more reliably then? I didn't even know about it... :smile:

Reply:
First of all it is faster. There is an automatic redirect from download.nue.suse.com to this mirror for some files as the mirror is selective. Seems to not work for some pieces which I need in the scale cloud. Probably fixing the redirect is the correct way to go but to be sure I would like this setting as it only applies to this particular cloud.

Reply:
Yes, that makes sense.